clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoAdminClient.java

```diff
@@ -157,6 +180,264 @@ public boolean dropMetalake(NameIdentifier ident) {
+   * @throws RuntimeException If adding the User encounters storage issues.
+   */
+  public User addMetalakeAdmin(String user) {
+    UserAddRequest req = new UserAddRequest(user);
+    req.validate();
+
+    UserResponse resp =
+        restClient.post(
+            String.format(API_ADMIN_PATH, ""),
+            req,
+            UserResponse.class,
+            Collections.emptyMap(),
+            ErrorHandlers.userErrorHandler());
+    resp.validate();
+
+    return resp.getUser();
+  }
+
+  /**
+   * Removes a metalake admin.
+   *
+   * @param user The name of the User.
+   * @return `true` if the User was successfully removed, `false` otherwise.
+   * @throws RuntimeException If removing the User encounters storage issues.
+   */
+  public boolean removeMetalakeAdmin(String user) {
+    RemoveResponse resp =
+        restClient.delete(
+            String.format(API_ADMIN_PATH, user),
+            RemoveResponse.class,
+            Collections.emptyMap(),
+            ErrorHandlers.userErrorHandler());
+    resp.validate();
+
+    return resp.removed();
+  }
+
+  /**
+   * Loads a Role.
+   *
+   * @param metalake The Metalake of the Role.
+   * @param role The name of the Role.
+   * @return The loading Role instance.
+   * @throws NoSuchRoleException If the Role with the given identifier does not exist.
+   * @throws RuntimeException If loading the Role encounters storage issues.
+   */
+  public Role loadRole(String metalake, String role) throws NoSuchRoleException {
+    RoleResponse resp =
+        restClient.get(
+            String.format(API_METALAKES_ROLES_PATH, metalake, role),
+            RoleResponse.class,
+            Collections.emptyMap(),
+            ErrorHandlers.roleErrorHandler());
+    resp.validate();
+
+    return resp.getRole();
+  }
+
+  /**
+   * Drops a Role.
+   *
+   * @param metalake The Metalake of the Role.
+   * @param role The name of the Role.
+   * @return `true` if the Role was successfully dropped, `false` otherwise.
+   * @throws RuntimeException If dropping the User encounters storage issues.
+   */
+  public boolean dropRole(String metalake, String role) {
+    DropResponse resp =
+        restClient.delete(
+            String.format(API_METALAKES_ROLES_PATH, metalake, role),
+            DropResponse.class,
+            Collections.emptyMap(),
+            ErrorHandlers.roleErrorHandler());
+    resp.validate();
+
+    return resp.dropped();
+  }
+
+  /**
+   * Creates a new Role.
+   *
+   * @param metalake The Metalake of the Role.
+   * @param role The name of the Role.
+   * @param properties The properties of the Role.
+   * @param securableObject The securable object of the Role.
+   * @param privileges The privileges of the Role.
+   * @return The created Role instance.
+   * @throws RoleAlreadyExistsException If a Role with the same identifier already exists.
+   * @throws RuntimeException If creating the Role encounters storage issues.
+   */
+  public Role createRole(
```

Comment:
BTW, the `createRole` server API has typo, please fix it.

Reply:
Done.